#**LABUBU**

---



Nama Anggota :

                   - MADE SURYA DARMAWAN RIJASA/  24083010119
                   - Daffa Muhammad khrisadi/ 24083010062
                   - Muhammad Jazuli / 2408301093
                   - Baihaqi Nur Muhammad / 24083010111
                   

              


**Penjelasan Toko**

---
data ini di-scraping melalu aplikasi tokopedia tepatnya di official store samsung. Nama filenya adalah Web Scrapping TOKPED Samsung dan pada bagian link_produk mengarahkan ke domain tokopediannya.



**Metode Scrapping**


---

scraping dilakukan langsung di halaman web tokopedia dengan menggunakan beutifulsoup dan selenium yang bertujuan untuk mengambil elemen seperti nama produk, harga produk, rating, jumlah penjualan, dll. Lalu file disimpan dalam bentuk csv, setelah itu dataset dibaca menggunakan import pandas menampilkan beberapa baris pertama dan baris terakhir untuk menampilkan strukur dan nama kolom, . Selanjutnya dilakukan perapihan dimana kolom yang tidak terlalu penting dihilangkan dan menyisakan kolom yang penting saja seperti nama_produk, harga_produk, rating, persentase diskon,harga sebelum diskon, jumlah produk yang terjual, lokasi, dan link produknya. setelah struktur terusun rapi dataset disimpan kembali berupa file baru sebagai hasil data yang sudah tersusun rapi dan bersih serta siap digunakan untuk analisis lebih lanjut. Hasil akhir dari proses ini yaitu data yang telah bersih dan terstruktur siap untuk dilakukan analisis dan divisualisasikan.

In [ ]:
import pandas as pd

file_path = 'Web Scrapping TOKPED Samsung - Web Scraper - Sheet1.csv'
df = pd.read_csv(file_path)
display(df)

,nama_produk,harga_sekarang,harga_sebelum_diskon,persentase_diskon,rating,jumlah_terjual,biaya_pengiriman,lokasi_penjual,link_produk
0,Samsung Microwave Grill 23L dengan Browning Pl...,Rp1.569.000,Rp1.669.000,6%,5.0,Terjual 100+•bintang 5 (22 rating),Kurir Toko Rp0,Kota Administrasi Jakarta Utara,https://www.tokopedia.com/samsung-official-sto...
1,"Samsung Microwave Oven Solo, 20L - MS20A3010ALSE",Rp799.000,Rp939.000,>15%,4.9,Terjual 500+•bintang 4.9 (63 rating),Kurir Toko Rp0,Kota Administrasi Jakarta Utara,https://www.tokopedia.com/samsung-official-sto...
2,Samsung Mesin Cuci WA80 Top Load Wobble Techno...,Rp2.729.000,Rp3.279.000,17%,4.9,Terjual 750+•bintang 4.9 (141 rating),Kurir Toko Rp0,Kota Administrasi Jakarta Utara,https://www.tokopedia.com/samsung-official-sto...
3,Samsung Kulkas 2 Pintu RT22 dengan Digital Inv...,Rp4.799.000,Rp5.939.000,19%,4.3,Terjual 28•bintang 4.3 (4 rating),Ongkir Rp264.000,Kota Administrasi Jakarta Utara,https://www.tokopedia.com/samsung-official-sto...
4,Samsung Soundbar HW-T420 2.1 Ch dengan Powerfu...,Rp1.589.000,Rp3.499.000,55%,4.9,Terjual 250+•bintang 4.9 (90 rating),Kurir Toko Rp0,Kota Administrasi Jakarta Utara,https://www.tokopedia.com/samsung-official-sto...
...,...,...,...,...,...,...,...,...,...
75,[ Bundling ] Samsung Galaxy S26+ [12/512] [Bla...,Rp21.748.000,Rp22.498.000,3%,NaN,NaN,NaN,NaN,https://www.tokopedia.com/samsung-official-sto...
76,[ Bundling ] Samsung Galaxy S26 Ultra [12/512]...,Rp27.498.000,Rp28.498.000,4%,NaN,1 barang berhasil terjual,Ongkir Rp23.000,Kota Administrasi Jakarta Utara,https://www.tokopedia.com/samsung-official-sto...
77,[ Bundling ] Samsung Galaxy S26 Ultra [16/1TB...,Rp30.498.000,Rp31.498.000,3%,NaN,4 barang berhasil terjual,Ongkir Rp23.000,Kota Administrasi Jakarta Utara,https://www.tokopedia.com/samsung-official-sto...
78,Samsung Air Conditioner Neo Alpha Inverter 1/2...,Rp2.899.000,Rp4.299.000,>33%,NaN,3 barang berhasil terjual,Kurir Toko Rp0,Kota Administrasi Jakarta Utara,https://www.tokopedia.com/samsung-official-sto...


**Penjelasan data**


---

Data ini adalah hasil scraping produk produk samsung yang di dapat dari marketplace Tokopedia. Data ini memiliki jumlah baris sebanyak 80 dan jumlah kolom sebanyak 9. Artinya terdapat 80 produk samsung yang dianalisis dengan 9 atribut informasi

Beberapa kolom memiliki nilai kosong karena tidak semua produk memiliki diskon, produk belum memiliki ulasan, atau belum ada penjualan signifikan. Berikut adalah penjelasan setiap kolom:
  1. nama_produk = Nama lengkap produk

1.   harga_sekarang = Harga jual saat ini.
2.   harga_sebelum_diskon = Harga asli sebelum diskon.
4.	persentase_diskon = Besar diskon dalam persen.
5.	rating = Nilai rating produk (skala 1–5).
6.	jumlah_terjual = Total produk terjual.
7.	biaya_pengiriman = Informasi ongkir/gratis ongkir.
8.	lokasi_penjual = Kota/wilayah toko.
9.	link_produk = URL produk.

sebagian besar kolom memiliki tipe data berupa string. Hanya ada 1 kolom bertipe float64 yaitu rating


**Penjelasan value di toko**


---
Dalam data yang kita miliki pasti terdapat "value" adalah sebuah isi dari setiap variabel pada data, misal kita ambil dari data=

Pada kolom harga → value berupa angka harga seperti Rp1.500.000

Pada kolom rating → value berupa angka desimal seperti 4.9

Pada kolom diskon → value berupa persentase seperti 25%

Value ini muncul karena web scraping bekerja dengan cara menangkap isi dari setiap elemen yang dipilih (selector) di dalam struktur halaman web. Jadi, setiap informasi yang tampil di halaman toko akan disimpan sebagai value dalam dataset.